src/h3/mod.rs

```diff
@@ -1410,6 +1423,13 @@ impl Connection {
         Ok(())
     }
 
+    /// Gets the raw settings from peer including unknown and reserved types.
+    ///
+    /// The order of settings is the same as received in the SETTINGS frame.
+    pub fn peer_settings_raw(&self) -> Option<Vec<RawSetting>> {
+        self.peer_settings.raw.clone()
```

Comment:
Could just return a slice here, rather than `Vec`, so the application can decide if it wants to clone or not.